### PR TITLE
SotA clarifications and prose edits.

### PR DESCRIPTION
**Pull request by max-torch**
This pull request provides solutions for #6120, #6119, #6121, #6123.

### CODE REVIEW

File: data/campaigns/Secrets_of_the_Ancients/scenarios/21_Against_the_World.cfg [hunk 1]
```diff
@@ -635,7 +635,7 @@ We finally made it out of the mountains. We crossed the Ford of Abez late this m
         [/message]
         [message]
             speaker=Ardonna
-            message= _ "That’s why getting through Knalga was so easy."
+            message= _ "That’s why getting through Knalga was so easy. By splitting their forces, they just allowed us to defeat them in detail."
```
Comment (stevecotton) [inline data/campaigns/Secrets_of_the_Ancients/scenarios/21_Against_the_World.cfg:638]: Sometimes, less is more. I think the original text is better, leaving it to the player to work out how the new information fits into the story.

File: data/campaigns/Secrets_of_the_Ancients/scenarios/02_Dark_Business.cfg [hunk 2]
```diff
@@ -255,6 +255,9 @@ I decided to hide in the cemetery. That way I could try my experiment to animate
             [note]
                 description= _ "This scenario takes place entirely at night."
             [/note]
+            [note]
+                description= _ "You will gain an advantage if the village labeled 'Guardhouse' is in your possession."
```
[the remark this answers is not in the document]
Comment (max-torch) [reply data/campaigns/Secrets_of_the_Ancients/scenarios/02_Dark_Business.cfg:259]: Would you be okay to instead add an extra explanation at the dialog that plays when the player captures the guardhouse:
image

[the remark this answers is not in the document]
Comment (max-torch) [reply data/campaigns/Secrets_of_the_Ancients/scenarios/02_Dark_Business.cfg:259]: ok so now there are two options to choose from: the additional sentence in the dialog that pops up when taking the guardhouse, the objective note, or both.

File: data/campaigns/Secrets_of_the_Ancients/scenarios/21_Against_the_World.cfg (hunk 1 above)
Comment (nemaara) [reply data/campaigns/Secrets_of_the_Ancients/scenarios/21_Against_the_World.cfg:638]: I think the original is fine as well.

**Pull request merged**